### PR TITLE
Snake

### PR DESCRIPTION
**Pull request by SuperBoyne**
This is a version of the game Snake made for the Sprig console.

It recreates the food, movement, score and progressive speeding systems, and a restart feature.

I'm Boyne Gregg, the guy who made it. I'm a 14 year old developer who has experience with Python, Java, JavaScript, HTML, CSS, SQL, and more. I'm part of an FRC robotics team, and am now mentoring an FTC robotics team that I used to be a part of.

### CODE REVIEW

**Review by LucasHT22 — COMMENTED** [review]
Thanks for PR! Wow, 14y

File: games/Snake.js [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+      gameOver();
+    }
+    i++;
+  })
+}
+
+let interval = setInterval(move, 400);
+
+afterInput(() => {
+  /* count the number of tiles with goals
```
Comment (LucasHT22) [inline games/Snake.js:295]: what is the reason for these commented codes?

Comment (SuperBoyne) [reply games/Snake.js:295]: Ah, it seems I accidentally left in some comments from the original block pushing game. I can submit a new pull request with those unnecessary lines removed.

**Comment by LucasHT22** [comment]
for me there is a bug, if you click "i" the game behaves strange

**Comment by SuperBoyne** [comment]
> for me there is a bug, if you click "i" the game behaves strange

That's intentional, i will reset the game to its initial state.

**Comment by LucasHT22** [comment]
> Thanks for PR! Wow, 14y

I'm 13y haha

**Comment by SuperBoyne** [comment]
No problem!

File: games/Snake.js (hunk 1 above)
Comment (LucasHT22) [reply games/Snake.js:295]: @SuperBoyne

Comment (LucasHT22) [reply games/Snake.js:295]: don't need a new PR! Just `files changed` >` ...` > `edit`

Comment (SuperBoyne) [reply games/Snake.js:295]: Alright, I'll do that, and add some music while I'm at it. :)

**Review by LucasHT22 — COMMENTED** [review]
Good changes! but for me, this is appearing this is showing up `Uncaught ReferenceError: Cannot access 'playback' before initialization on line 260 in column 3. Open the browser console for more information.`

**Comment by SuperBoyne** [comment]
Ah, yes, I made a mistake. I've fixed it and everything should be working as normal.

**Comment by LucasHT22** [comment]
Very good!

**Comment by LucasHT22** [comment]
Looks good to me:) Congrats!
you can get your sprig here! https://sprig-order.hackclub.dev/

**Comment by SuperBoyne** [comment]
Thank you so much! Have a nice day!

**Pull request merged**